### PR TITLE
overview page of cloud deployment

### PR DESCRIPTION
**Pull request by patriknw**
I expect more content from PM on this, but this can be a starting point of technical content.

### CODE REVIEW

File: docs-source/docs/modules/deployment/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+The easiest way to deploy and manage Akka Microservices in Kubernetes is to use the Akka Platform Operator. You get access to it from the https://aws.amazon.com/marketplace[Amazon Marketplace {tab-icon}, window="tab"] and install it in your Amazon EKS cluster.
+
+FIXME direct link ^ to product page and better product pitch.
+
+The Akka Platform Operator provides the following high level features:
+
+* convenient deployment without having to define detailed Kubernetes manifests
+* integration with Akka Cluster Bootstrap
+* integration with Akka Management health checks
+* integration with relational databases and Cassandra
```
Comment (octonato) [inline docs-source/docs/modules/deployment/pages/index.adoc:14]:
```suggestion
* integration with Relational Databases and Cassandra
```

**Review by chbatey — APPROVED** [review]
Looks good

File: docs-source/docs/modules/deployment/pages/index.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+== Overview of deployment procedure
+[#deployment_procedure]
+
+=== Deploying an Akka Microservices application
+
+image::deployment-overview1.png[Deployment 1]
+
+. A https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Kubernetes Custom Resource Definition {tab-icon}, window="tab"] named `akkamicroservices.akka.lightbend.com` defines the deployment descriptor resource for Akka Microservices applications that that the Akka Operator will deploy.
+. The Akka Operator is installed and is always running in the Kubernetes cluster.
+. A Docker image of the Akka Microservices application is built and published with Maven or sbt.
```
Comment (andreaTP) [inline docs-source/docs/modules/deployment/pages/index.adoc:28]: I think we can remove `with Maven or sbt`

File: docs-source/docs/modules/deployment/pages/index.adoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+. The deployment descriptor for the Akka Microservice is defined in a yaml file. It refers to the Docker published image and specifies other operational properties. It is applied to the Kubernetes cluster with `kubectl` (Kubernetes standard command-line tool).
+. The operator sees the new deployment descriptor and creates a https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Kubernetes Deployment {tab-icon}, window="tab"]
+. The container in the Pod downloads the application image and starts the Akka Microservices application in the JVM using additional configuration provided by the Akka Platform Operator.
```
Comment (andreaTP) [inline docs-source/docs/modules/deployment/pages/index.adoc:31]: This sentence is somehow involved.
Something like:
```
The Akka Platform Operator automatically provides additional configuration to the Akka Microservices application Pod
```
wdyt? no need to go into detail at this level IMHO

File: docs-source/docs/modules/deployment/pages/index.adoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+When updating the Akka Microservice the steps 3 - 6 are repeated. The Akka Platform Operator sees the changes to the deployment descriptor and updates the Kubernetes Deployment accordingly. Kubernetes performs a rolling update of the application.
```
Comment (andreaTP) [inline docs-source/docs/modules/deployment/pages/index.adoc:33]:
```suggestion
When updating the Akka Microservice application the steps 3 - 6 are repeated. The Akka Platform Operator sees the changes to the deployment descriptor and updates the Kubernetes Deployment accordingly. Kubernetes performs a rolling update of the application.
```

**Review by andreaTP — APPROVED** [review]
minor comments and LGTM

**Review by octonato — APPROVED** [review]

File: docs-source/docs/modules/deployment/pages/aws-ingress.adoc [hunk 5]
```diff
@@ -3,7 +3,7 @@
 
 include::partial$include.adoc[]
 
-To expose gRPC or HTTPS to the internet you can ues the https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/[AWS Load Balancer Controller {tab-icon}, window="tab"] with a https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes Ingress {tab-icon}, window="tab"] created by the Akka Platform operator.
+To expose gRPC or HTTPS to the internet you can ues the https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/[AWS Load Balancer Controller {tab-icon}, window="tab"] with a https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes Ingress {tab-icon}, window="tab"] created by the Akka Platform Operator.
```
Comment (octonato) [inline docs-source/docs/modules/deployment/pages/aws-ingress.adoc:6]:
```suggestion
To expose gRPC or HTTPS to the internet you can use the https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/[AWS Load Balancer Controller {tab-icon}, window="tab"] with a https://kubernetes.io/docs/concepts/services-networking/ingress/[Kubernetes Ingress {tab-icon}, window="tab"] created by the Akka Platform Operator.
```

**Comment by patriknw** [comment]
incorporated feedback: e3d84c4

**Pull request merged**